content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,707 @@
+
+Below this message are five avatar thumbnails to choose from. If an avatar has already been assigned
+to a referral, the thumbnail is greyed out, and the name the avatar has been chosen for appears
+beneath the avatar. Click on an available avatar to select it.
+
+If more than one referral was hired since opening the Referrals application, after selecting the
+first avatar, the user is prompted to select another avatar for the subsequent hired referral. Once
+all avatars have been selected, the dashboard loads and all the avatars are now visible. Mouse over
+each avatar and their name is displayed above them.
+
+.. image:: referrals/avatars.png
+   :align: center
+   :alt: The hired screen. A selection of avatars are presented to chose from, with any already
+         chosen are greyed out.
+
+Referral points
+===============
+
+The referral program is based on earning points that are then exchanged for rewards. On the main
+Referrals application dashboard, the top displays a summary of the user's current points. The left
+side of the summary displays the total points earned, and the right side displays the points that
+are available to spend.
+
+To see the various referrals and the points earned for each, click on one of the buttons directly
+beneath the avatars. The options are :guilabel:`Referrals`, :guilabel:`Ongoing`, and
+:guilabel:`Successful`.
+
+The current number of referrals who are still active in the recruitment pipeline, and have not yet
+been hired or refused, appear above the :guilabel:`Ongoing` button. The number of referrals who have
+been hired appear above the :guilabel:`Successful` button. The total number of referrals (both the
+ongoing and successful referrals combined) appear above the :guilabel:`Referrals` button.
+
+.. note::
+   To view the referrals associated with each of the categories, :guilabel:`Referrals`,
+   :guilabel:`Ongoing`, and :guilabel:`Successful`, click on the corresponding button. All the
+   referrals for that specific category are displayed.
+
+My referrals
+------------
+
+To see all the referrals (both ongoing and successful) click the :guilabel:`Referrals` button. A
+:guilabel:`My Referral` window loads. This screen displays all the referrals, with each referral in
+its own referral card.
+
+A successful referral has a green line down the left side of the card, and displays a `✓ Hired`
+image in the top right corner of the card. Referrals that are in process display an `In Progress`
+image in the top right corner.
+
+Each referral card lists the name of the applicant, the department the job position is in, the name
+of the job position, the person responsible for filling the role (recruiter), and the points earned.
+
+Points
+~~~~~~
+
+The points that can be earned for a referral are the same across all job positions. Each stage of
+the recruitment process has corresponding points assigned to it. The stages listed correspond to the
+stages configured in the *Recruitment* application.
```

Comment:
Only through recruitment- you can't do anything to change any points in Referrals alone. That's why I linked to the recruitment doc (which isn't published yet) because there it explains how to change points.